charm/ksql/tests/00-setup

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Let's remove the generated `tests/` directory. In my opinion, amulet is more trouble than it's worth and easily replaced with a shell script.